web/src/layout/leftpanel/zonedetailspanel.js

```diff
@@ -26,7 +26,7 @@ const mapStateToProps = state => ({
 
 const SocialButtons = styled.div`
   @media (max-width: 767px) {
-    display: ${location.pathname !== '/map' ? 'none !important' : 'block' };
+    display: ${props => props.pathname !== '/map' ? 'none !important' : 'block' };
```

Comment:
You need to also add this `pathname` prop to `<SocialButtons />` :)

Reply:
Sorry about that one. Didn't notice.